src/parser/jobs/ast/modules/Sect/Sect.tsx

```diff
@@ -142,8 +142,12 @@ export default class Sect extends Module {
 	}
 
 	private onCast(event: CastEvent) {
-		// If they casted a sect at anytime it means they pulled without one on
-		this.pullWithoutSect = true
+		if (event.timestamp < this.parser.eventTimeOffset) {
+			// This was before the pull and likely added by the PrecastStatus module.
```

Comment:
blank if section? Is there supposed to be logic here?

Reply:
It was there just to document why the check was necessary.

Reply:
I still find the blank logic confusing - code shouldn't be a substitute for comments, especially when it leads to noop-style logic to have 'self documenting code'. Instead of using a blank `if` block, document as part of a comment and invert the logic to be:

```js
if (event.timestamp >= this.parser.eventtimeOffset) {
```

Reply:
Fixed this in #1023, since that's the PR those changes came from.

Reply:
I'll let this go, but in the future document when PRs have ordered dependencies, and please try not to base your PR off another PR branch - small atomic commits are much easier to reason about.

Reply:
I did say in the PR description that it was branched off of #1023..

Reply:
I understand this was branched off of #1023, but I didn't know about the ordered dependency until we discussed it., and the suggestion was to base your PRs off of master moving forward (unless there's a compelling reason not to).

Reply:
Shouldn't be an issue since this was based off of that to avoid duplicating/conflicting adding myself as a contributor, and I didn't add it to that one so I could keep the individual PR small. This was a bit of a one-off situation for me.